README.md

```diff
@@ -1 +1,25 @@
-# angular-ngroute-lab
\ No newline at end of file
+# Integrate ngRoute
+
+## Objectives
+
+- Setup ng-view with our application
+- Fetch data from the stubbed API
+- Change view using ngRoute
+
+## Instructions
+
+Run `npm install` in your console to install dependencies, and then run `node server.js` to run the local web server. In another console, run `stubby -d config.yml` (this will run our mocked backend).
+
+Our mocked backend has three URLs -
+
+http://0.0.0.0:8882/rest/user/liam
+http://0.0.0.0:8882/rest/user/jayden
+http://0.0.0.0:8882/rest/user/mary
+
+This will return data on the given user.
+
+Add `ngRoute` to our app and then configure the route for `/user/:name`, using `views/user.html` for the view, and `UserController` for the user.
```

Comment:
Add a specific instruction for how the view should be formatted ie an h2 for the first and last name, ul for address, etc.